docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: "3"
+
+x-jellyfish-template: &jellyfish-template
+  build: .
+  environment: &jellyfish-environment
+    ERLANG_COOKIE: "panuozzo-pollo-e-pancetta"
+    SERVER_API_TOKEN: "development"
+    SECRET_KEY_BASE: "super-secret-key"
+    VIRTUAL_HOST: "localhost"
+    NODES: "app@app1 app@app2"
+  networks:
+    - net1
+  restart: on-failure
+  healthcheck:
+    test:
+      [
+        "CMD",
+        "sh",
+        "-c",
+        "curl --fail -H \"authorization: Bearer $$SERVER_API_TOKEN\" http://localhost:$$PORT/room || exit 1"
+      ]
+    interval: 3s
+    retries: 2
+    timeout: 2s
+    start_period: 15s
+
+services:
+  test:
+    image: membraneframeworklabs/docker_membrane
+    command:
+      - sh
+      - -c
+      - |
+        cd app/ 
+        mix deps.get 
+        MIX_ENV=test_containerised mix test --only containerised
+    volumes:
+      - .:/app
+      - /app/_build
+      - /app/deps
```

Comment:
Why those volumes?

Reply:
As we run it on `docker_membrane,` project files are not included. Then we exclude `_build` and `deps` directory because it could be compiled with different flag etc.